### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by drhaynes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,81 @@
...
 
+### Getting started
+
+At the currently time of writing, WG-Maply does not yet support Carthage (see [Add Carthage support](https://github.com/mousebird/WhirlyGlobe/issues/542) issue in main WG-Maply repo). For this reason, manual building of the framework is required. 
+
+Steps to do this are outlined below:
+
+1. Clone this repo
+2. Carthage update (this brings in OSTransformation framework): `$ carthage update --platform iOS --no-use-binaries `
+3. Submodule init (this brings in WG-Maply): `$ git submodule update --init`
```
Comment (dhardiman) [inline README.md]: If this was `git submodule update --init --recursive` you probably wouldn't need steps 4 and 5. Your submodule reference should already be the 2.4.1_beta3 tag anyway.

File: README.md [hunk 2]
```diff
@@ -2,5 +2,81 @@
...
+
+	- (void)addMapToViewHierarchy {
+	    [self.view addSubview:self.maplyViewController.view];
+	    [self.view sendSubviewToBack:self.maplyViewController.view];
+	    [self addChildViewController:self.maplyViewController];
+	}
+	```
+
+
+10. Now, link against `libWhirlyGlobe-MaplyComponent.a`, and its associated framework deps:
```
Comment (dhardiman) [inline README.md:54]: Can libWhirlyGlobe-MaplyComponent.a not be statically linked in to our framework?

Comment (drhaynes) [inline README.md:54]: Further investigation is needed on how to do this. Specifying it the link build phase isn't sufficient to statically link it.

Comment (drhaynes) [inline README.md:54]: https://github.com/OrdnanceSurvey/os-maply-adapter-ios/issues/8

**Comment by dhardiman** [comment]
LGTM when the build passes. Unless the build failure is a known issue, in which case carry on...

**Comment by drhaynes** [comment]
Hoping it will pass this time...

**Comment by drhaynes** [comment]
Coverage is skewed as it's including system framework headers. To investigate in a subsequent PR.

**Pull request merged**